template/web/wp-content/themes/<%= slug %>/app/Repositories/PostRepository.php

```diff
@@ -7,10 +7,24 @@
 
 namespace Studiometa\Repositories;
 
+use Timber\PostQuery;
+
 /** Class */
-class PostTypeRepository extends Repository {
+final class PostRepository extends Repository {
```

Comment:
Maybe this should not be a final class so it can be extended for custom post types? 🤔